calico-3.30.yaml

```diff
@@ -686,6 +688,189 @@ subpackages:
           runs: |
             typha-client --help
 
+  - name: "calico-goldmane-${{vars.short-package-version}}"
+    description: "Goldmane"
+    dependencies:
+      provides:
+        - calico-goldmane=${{package.full-version}}
+      runtime:
+        - glibc
+        - merged-bin
+        - merged-usrsbin
+        - wolfi-baselayout
+    pipeline:
+      - uses: go/build
+        with:
+          modroot: .
+          packages: ./goldmane/cmd
+          output: goldmane
+      - uses: go/build
+        with:
+          modroot: .
+          packages: ./goldmane/cmd/flowgen
+          output: flowgen
+      - uses: go/build
+        with:
+          modroot: .
+          packages: ./goldmane/cmd/stream
+          output: stream
+      - uses: go/build
+        with:
+          modroot: .
+          packages: ./goldmane/cmd/health
+          output: health
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/
+          ln -sf /usr/bin/goldmane "${{targets.subpkgdir}}"/goldmane
+          ln -sf /usr/bin/health "${{targets.subpkgdir}}"/health
+          ln -sf /usr/bin/flowgen "${{targets.subpkgdir}}"/flowgen
+          ln -sf /usr/bin/stream "${{targets.subpkgdir}}"/stream
```

Comment:
Why are we linking into `/` here?

Reply:
In the upstream Dockerfiles, they also symlink the files to `/`.

Reply:
I'd like to see at least a comment to that effect.

AFAICT nothing else in this package is symlinked to `/`: do we know why they do that?